interp/src/primitives/stateful.rs

```diff
@@ -33,25 +20,27 @@ pub struct StdMultPipe {
     pub product: Value,
     update: Option<Value>,
     queue: VecDeque<Option<Value>>, //invariant: always length 2.
+    signed: bool,
```

Comment:
Not saying we should do this but it looks like a good place to potentially use [const generics](https://blog.rust-lang.org/2021/02/26/const-generics-mvp-beta.html)

Reply:
That is very cool and could definitely work here!